packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -195,7 +200,31 @@ const chevronStyles = style({
   flexShrink: 0
 });
 
+
 function DisclosureHeader(props: DisclosureHeaderProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    UNSAFE_className,
+    UNSAFE_style,
+    children
+  } = props;
+  let domRef = useDOMRef(ref);
+  return (
+    <div
+      style={UNSAFE_style}
+      className={(UNSAFE_className ?? '') + style({display: 'flex', alignItems: 'center', gap: 4})}
+      ref={domRef}>
+      {children}
```

Comment:
An option for shorthand: if `typeof children === 'children'`, we could automatically wrap `children` in a `DisclosureTrigger`, which would let you use our existing API as well.

No need to implement this now, we could discuss it later